docs/README.md

```diff
@@ -0,0 +1,31 @@
+# Building the docs
+
+1. Clone repo checking out submodules
+
+```shell
+git clone git@github.com:CQCL/Qermit.git --recurse-submodules
+```
+
+2. Install Qermit and docs dependencies
+
+```shell
+python -m venv ".venv"
+source .venv/bin/activate
+pip install -e .[docs] 
+```
```

Comment:
I think we could remove the reference to venv and just allow the user to use whichever means they prefer.
You could also mention at this point that poetry is supported.